local/kaltura/service.php

```diff
@@ -98,6 +99,7 @@
     'title' => $title,
     'metadata' => $metadata,
     'editor' => $editor,
+    'previewltilauncher' => $previewltilaunchurl->out(),
```

Comment:
because its fucking hard to read variables thisiswhy (or better yet - thisIsWhy)